frontend/src/main/pages/SectionSearch/SectionSearchIndexPage.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import SectionSearchForm from "main/components/SectionSearch/SectionSearchForm";
+import { useBackendMutation } from "main/utils/useBackend";
+// import SectionsTable from "main/components/SectionSearch/SectionSearchTable";
```

Comment:
Please remove commented out code until you need it; especially for an import, which is easy to restore.